src/sz390.java

```diff
@@ -6009,7 +6251,9 @@ private void exec_test_cmd(){
 	    tz390.put_trace("  B=addr      set base for rel addr (ie B=15r% sets base to (r15) 24 bit");
 	    tz390.put_trace("  D           display DCB file status, DDNAME, and DSNAME information");
 	    tz390.put_trace("  E           toggle EBCDIC/ASCII mode for dumping storage etc.");
-	    tz390.put_trace("  F nn        display specified floating point registers else all F0-FF");
+	    tz390.put_trace("  F nn        display specified floating-point register else FPC and all F0-FF"); // #515
```

Comment:
I find the reference F0-FF confusing as it seems to be hex numbers. I would like to clarify by changing F0-FF to FPR 0-F